src/libraries/arrays/AddressArrays.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.4.13;
+
+
+library AddressArrays {
+    /*
+     * @dev Returns a slice of the array
+     * @param self The array to make a slice from.
+     * @param start The index to start at.
+     * @param size The target number of elements to return in the slice
+     * @return A newly allocated slice containing the requested elements
+     */
+    function slice(address[] _self, uint256 _start, uint256 _size) internal returns (address[]) {
```

Comment:
Ooh, good use of extension methods.  I like it.  Recommend making sure we have adequate test coverage, as this is one of those areas of code that people will just assume works correctly.